src/Actions/DownloadAsset.php

```diff
@@ -20,7 +21,11 @@ public function visibleTo($item)
 
     public function visibleToBulk($items)
     {
-        return false;
+        if ($items->whereInstanceOf(Asset::class)->count() !== $items->count()) {
+            return false;
+        }
+
+        return true;
```

Comment:
I think this whole method can just be removed now. Unless there was a specific reason you had added it like this?

Reply:
I think I copied that from somewhere else not realising it wasn't necessary here. Removed.